### PR TITLE
Update threat-trackers.md

### PR DESCRIPTION
**Pull request by ajaj-shaikh**
Added "Security Reader" role as this is by design available for sec reader role

### CODE REVIEW

File: microsoft-365/security/office-365-security/threat-trackers.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ The new Noteworthy threat filter highlights items that were recently detected th
 
 - If your organization doesn't already have these Office 365 Threat Investigation and Response capabilities, see [How do we get Office 365 Threat Investigation and Response capabilities?](office-365-ti.md).
 
-- Make sure that your security team has the correct roles and permissions assigned. You must be a global administrator, or have the Security Administrator or Search and Purge role assigned in the Microsoft 365 Defender portal. See [Permissions in the Microsoft 365 Defender portal](mdo-portal-permissions.md).
+- Make sure that your security team has the correct roles and permissions assigned. You must be a global administrator, or have the Security Administrator, or have the Security Reader or Search and Purge role assigned in the Microsoft 365 Defender portal. See [Permissions in the Microsoft 365 Defender portal](mdo-portal-permissions.md).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/threat-trackers.md]:
```suggestion
- Make sure that your security team has the correct roles and permissions assigned. You must be a global administrator, or have the Security Administrator, Security Reader, or Search and Purge role assigned in the Microsoft 365 Defender portal. See [Permissions in the Microsoft 365 Defender portal](mdo-portal-permissions.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ajaj-shaikh, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ajaj-shaikh** [comment]
@denisebmsft @MSFTTracyP can you please merge this pull request?

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder.

**Pull request merged**